Wikimate.php

```diff
@@ -346,6 +354,57 @@ protected function token($type = self::TOKEN_DEFAULT)
         }
     }
 
+    /**
+     * Obtains parameter lists for API modules.
+     * For specific API modules where supported parameters vary with
+     * MediaWiki versions, fetch and store these parameter lists for
+     * use by the API calls of those modules.  Current modules list:
+     * - logout
```

Comment:
```suggestion
     * Obtains parameter lists for API modules.
     * For specific API modules where supported parameters vary with
     * MediaWiki versions, this method is used to fetch and store
     * these parameter lists for use by the API calls of those modules.
     * Current list of methods that make use of this:
     * - logout
```

Reply:
I worded the last sentence slightly different: "Current list of modules that employ this mechanism:"

Reply:
My question is whether we want to refer to (Wikimate) methods or (MediaWiki API) modules. It seemed that the former makes more sense, but maybe I'm misreading your intention?

Reply:
Got it. Do you think it might be worth linking "logout" to https://www.mediawiki.org/wiki/API:Logout?

And also, how about rephrasing slightly to "Current list of modules for which this mechanism is employed"? I'm asking because who _employs_ the mechanism is Wikimate, while the API _provides_ the mechanism for use in all modules (IIUC).